admin/controllers/tech-support.js

```diff
@@ -492,11 +493,11 @@ const controller = {
   },
 
   postCheckResultsResyncAll: async function postCheckResultsResyncAll (req, res, next) {
-    res.locals.pageTitle = 'Resync All Results'
+    res.locals.pageTitle = 'Resync Uncalculated Results'
```

Comment:
It's not really "uncalculated" though is it?  More like 'Un-synchronised".

Even better, as the role is tech support here, perhaps something like "Synchronise new checks from Table Storage to Azure SQL".  


If the checkbox is ticked then the historical stuff is resynchronised too.

Reply:
yeah doesn't need to be high level, and makes more sense to us.
I've changed the checkbox to `Include checks that were previously synchronised`